### DISCUSSION TITLE
questions surrounding hwaccel and gpu usage

### DISCUSSION THREAD

**Discussion by finnaGIT** [blakeblackshear/frigate#15434] [Ask A Question] [answered]
Want to start by saying that I'm really grateful for the contributors and community here. I'm not a developer but am getting comfortable around linux and deployment/administration of services locally.

# Question
What specific processes are using my iGPU given my setup? (I've got 10% iGPU usage indicated in Frigate)

## My setup
i5-9500 running proxmox, and frigate is deployed in a Docker LXC. In a separate LXC, Scrypted is running. I've got 3 cameras currently and will expand to 6. My frigate configuration has go2rtc setup to grab the rtsp links from Scrypted. All cameras are encoded on-device H.264.

## Additional Discussion
Recognizing this may be a lapse in my knowledge (please educate me) - it's not clear where there would be any encoding/decoding/transcoding occurring at any of these interfaces contributing to the use of the iGPU. i start and end with h264.

Thanks for any and all feedback

**Comment by NickM-27** [comment, marked as answer]
Frigate decodes the streams to run motion and object detection on, every camera defined in frigate will have decoding occurring.

**Reply by finnaGIT** [reply]
thanks.

are you aware of any best practices when deploying frigate for object detection & nvr alongside scrypted (really only for homekit integration)?

am i doing something silly by bringing the restream from scrypted into frigate?

**Reply by NickM-27** [reply]
No, that seems fine
